docs/library/rp2.rst

```diff
@@ -35,8 +35,10 @@ For running PIO programs, see :class:`rp2.StateMachine`.
     - *out_init* configures the pins used for ``out()`` instructions.
     - *set_init* configures the pins used for ``set()`` instructions. There can
       be at most 5.
-    - *sideset_init* configures the pins used side-setting. There can be at
-      most 5.
+    - *sideset_init* configures the pins used for ``.side()`` modifiers. There
+      can be at most 5.
+    - *side_pindir* configures whether ``.side()`` modifiers are used for
+      pin directions instead of pin values.
```

Comment:
It's not clear which values are allowed for this argument.  Suggest something like:
```
*side_pindir* when set to ``True`` configures ``.side()`` modifiers to be used for pin directions, instead of pin values (the default, when ``False``).
```

Reply:
Looks good now, thanks.